pkglistgen/tool.py

```diff
@@ -743,6 +745,22 @@ def update_and_solve_target(
 
         file_utils.multibuild_from_glob(release_dir, '*.spec')
         self.build_stub(release_dir, 'spec')
+
+        todo_spec_files = []
+        package = Package(release_dir)
+        if package.get_status():
```

Comment:
This looks a bit weird. Is that really the only way in the osc API to figure out if there are changes?

It might have to filter out `' '` for unchanged files.

Reply:
There are many ways to see if there are changes. But this seemed a straight forward way to me. By my reading get_status returns an empty list [] if there are no changes, and otherwise one tuple of (status, filename) per changed file.

Reply:
So where would there be a space to filer out?

Reply:
FWICT it returns `(' ', filename)` if unchanged and not part of `exclude_states`

Reply:
osc itself uses `p.get_status(False, ' ')` in various places

Reply:
Ah. Changed.